node/service/src/rpc.rs

```diff
@@ -213,6 +213,7 @@ where
 			Arc::clone(&block_data_cache),
 			fee_history_cache,
 			fee_history_limit,
+			10, // TODO: sensible value (this came from frontier's node template)
```

Comment:
cc @tgmichel / @nanocryk what is a good value here? Comes from https://github.com/paritytech/frontier/pull/799

Reply:
10x was what we wanted to match Infura, so looks good to me.